docs-client/src/containers/MethodPage/GraphqlRequestBody.tsx

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useCallback, useEffect, useReducer, useState } from 'react';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import loadable from '@loadable/component';
+
+import {
+  buildClientSchema,
+  GraphQLSchema,
+  getIntrospectionQuery,
+} from 'graphql';
+import TextField from '@material-ui/core/TextField';
+import jsonPrettify from '../../lib/json-prettify';
+import { docServiceDebug } from '../../lib/header-provider';
+
+const graphqlPlaceHolder = `{
+  field(arg: "value") {
+    subField
+  }
+}`;
+const jsonPlaceHolder = jsonPrettify('{"foo":"bar"}');
+
+const QueryEditor = loadable(() => import('./QueryEditor'));
+
+interface Props {
+  requestBodyOpen: boolean;
+  requestBody: string;
+  onEditRequestBodyClick: React.Dispatch<unknown>;
+  onDebugFormChange: (value: string) => void;
+  schemaUrlPath: string;
+}
+
+const toggle = (prev: boolean, override: unknown) => {
+  if (typeof override === 'boolean') {
+    return override;
+  }
+  return !prev;
+};
+
+const parseJson = (requestBody: string) => {
```

Comment:
nit; parameter name can be more accurate/expressive (since there is already a variable called `requestBody`)

```suggestion
const parseJson = (s: string) => {
```